src/components/ProjectCard/ProjectCard.css

```diff
@@ -0,0 +1,215 @@
+.ProjectsCard {
+    width: 15rem;
+}
+  
+.ProjectsCard:hover {
+    transition: 0.5s;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    transform: scale(0.95);
+}
+.ProjectImageDiv{
+    margin: 0;
+    width: 100%;
+    height: 150px;
+    background-color: #008ac1;
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.BottomContainer {
+    padding: 10px;
+    height: 80px;
+}
+
+.ProjectsCardName{
+    width: 100%;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+    padding-top: 5px;
+}
+.ProjectsCardDesc{
+    width: 100%;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    color: #000000;
+    padding-top: 10px;
+
+}
+
+.ProjectDropDown {
+    position: relative;
+    display: inline-block;
+    z-index: 0;
+    margin-left: 70%;
+    margin-top: 20%;
+}
+  
+.ProjectDropDownContent {
+    display: block;
+    position: absolute ;
+    background-color: #f1f1f1;
+    min-width: 110px;
+    right: 0;
+    box-shadow: 0px 4px 8px 0px rgba(0,0,0,0.2);
+    z-index: 1;
+}
+  
+.ProjectDropDownContent div {
+    color: black;
+    padding: 0.5rem 1rem;
+    text-decoration: none;
+    display: block;
+    font-size: 13px;
+    text-align :left;
+}
+
+.ProjectDropDownContent div::first-letter {
+    text-transform: uppercase;
+}
+
+  
+.ProjectDropDownContent div:hover {background-color: #ddd;}
+
+.ProjectDropDown:hover .DropDownImg {
+    transition: 0.5s;
+    opacity: 0.8;
+    transform: scale(1.02);
+}
+
+
+/*///////////// Delete Model //////////////////*/
+
+.ProjectDeleteModel{
+    position: fixed;
+    display: hidden!important;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 99999;
+    background-color: rgba(0,0,0,0.5)
+}
+
+.ProjectDeleteModel .ModalParentWrap{
+    background-color: inherit;
+}
+
+.ProjectDeleteModel .ModalChildWrap{
+    width: 20rem;
+    height: unset;
+    min-height: 10rem !important;
+    z-index: 9999;
+    background-color: #f1f1f1;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+}
+
+.DeleteDescription{
+    cursor: pointer;
+}
+
+.DeleteProjectModelResponses{
+    display: grid;
+    height: 100%;
+    grid-gap: 1rem;
+    gap: 1rem;
+    grid-template-columns: repeat(2, auto);
+    align-items: center;
+    width: 100%;
+    margin-top: 4rem;
+}
+
+.DeleteProjectModelResponses button{
+    display: inline;
+}
+
+.DeleteMessageDiv{
+    margin: 1.5rem;
+    text-align: center;
+}
+.DeleteErrorDiv{
+    color: red;
```

Comment:
i suggest you use color codes for color: e.g `#f00` for red, not names

Reply:
This has been worked on. @mrwagaba